### ISSUE TITLE
Bug Report: Visual Loading Bar Glitch

### ISSUE THREAD

**Issue by JVNpixels** [FunkinCrew/Funkin#3121]
### Issue Checklist

- [X] I have properly named the issue
- [X] I have checked the issues/discussions pages to see if the issue has been previously reported

### What platform are you using?

Itch.io (Downloadable Build) - Windows

### If you are playing on a browser, which one are you using?

None

### Version

0.4.1

### Context (Provide images, videos, etc.)

# Glitch:
![image](https://github.com/user-attachments/assets/9f77e3bb-f8e7-4eb8-a24b-e4b4f0e47f10)
**Basically, if you open Friday Night Funkin' in fullscreen mode, it will screw up the loading bar**

### Steps to reproduce (or crash logs, errors, etc.)

1. Open Friday Night Funkin' in fullscreen
2. You should see that the bar is screwed up.

**Comment by Hundrec** [comment]
How were you able to open Funkin' in fullscreen?
The closest I could get to this was pressing Alt + Enter immediately after opening the .exe file

![image](https://github.com/user-attachments/assets/8126cdd0-e6ca-43b7-b409-99791a8d1288)

**Comment by JVNpixels** [comment]
![image](https://github.com/user-attachments/assets/e1285236-eac5-4ccf-a1e0-76b235432373)
![image](https://github.com/user-attachments/assets/92e70467-5d31-4803-a885-097e6b4830ef)
![image](https://github.com/user-attachments/assets/352f4b37-f24e-4185-b662-5c140b7a8ef6)

**Comment by JVNpixels** [comment]
still present in 0.5.2 also i dont feel like typing rn

**Comment by Hundrec** [comment]
Ah, that makes sense!
I hope they add an option for this suggestion https://github.com/FunkinCrew/Funkin/issues/2386 sometime in the future.

**Comment by NotHyper-474** [comment]
Solved in 0.7.x, presumably when they also fixed it being broken on mobile.

**Comment by JVNpixels** [comment]
Alright then, this issue will be closed.
